ansible_collections/arista/avd/roles/eos_designs/templates/facts/node_type/main.j2

```diff
@@ -23,6 +23,12 @@ switch:
 {# switch.default_evpn_role #}
   default_evpn_role: {{ ns.node_type_key_data.default_evpn_role | arista.avd.default('none') }}
 
+{# switch.default_underlay_routing_protocol #}
+  default_underlay_routing_protocol: {{ ns.node_type_key_data.default_underlay_routing_protocol | arista.avd.default('ebgp') }}
```

Comment:
Need to add ```default_underlay_routing_protocol``` in [node_type_keys](https://avd.sh/en/devel/roles/eos_designs/doc/node-types.html) document

Reply:
Same for mpls_lsr

Reply:
fixed